source/enumerate-collections.rst

```diff
@@ -250,6 +250,9 @@ All methods:
 
 Getting Collection Names
 ~~~~~~~~~~~~~~~~~~~~~~~~
+Drivers MAY implement a MongoClient method that returns an iterable of strings,
```

Comment:
Other parts of the document have a blank line after the heading underline. Should we have that here as well (if only for consistency)?